docs/dotnet/core/installation.md

```diff
@@ -20,3 +20,29 @@ Or, add via `<PackageReference>`
 <PackageReference Include="Microsoft.AutoGen.Contracts" Version="0.4.0-dev.1" />
 <PackageReference Include="Microsoft.AutoGen.Core" Version="0.4.0-dev.1" />
 ```
+
+# Additional Packages
+
+The *Core* and *Contracts* packages will give you what you need for writing and running agents using the Core API within a single process. 
+
+- *Microsoft.AutoGen.AgentChat* - An implementation of the AgentChat package for building chat-centric agent orchestration on top of the Core SDK
+- *Microsoft.AutoGen.Agents* - a package that has a small number of default agents you can use. 
+- *Microsoft.AutoGen.Extensions* - Extensions to support closely related projects including Aspire, Microsoft.Extensions.AI, and Semantic Kernel
+
+```sh
+dotnet add package Microsoft.AutoGen.AgentChat --version 0.4.0-dev-1
```

Comment:
The current version is 0.4.0-dev.1 ("-" => "."); we should just ensure we do a push of the docs immediately after release .2, I think.